src/components/pages/currentLocalWeather/currentLocalWeather.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
I'm getting the following error when I try to run your application: 
![image](https://user-images.githubusercontent.com/8356215/51218283-5001f000-18f1-11e9-96d2-de90cb9727e8.png)
I think you need to change this file name.